app.js

```diff
@@ -49,31 +49,39 @@ bottomDropdown.addEventListener('change', () => {
     // get the value of the bottom dropdown
     const value = bottomDropdown.value;
     // increment the bottom change count state
-    timesBottomChanged++;
+    bottomCount++;
+
     // update the dom for the bottom (NOTE use style.backgroundImage on the bottomEl div instead of trying to set the .src -- it's NOT an img tag!)
-    bottomEl.style.backgroundImage = `url("./assets/${value}-bottom.png")`;
+    bottomEl.style.backgroundImage = `url("./assets/${value}-pants.png")`;
     // update the stats to show the new count (call displayStats() to do this work)
     displayStats();
 });
 
 catchphraseButton.addEventListener('click', () => {
     // get the value of the catchphrase input
-    
+    console.log('Words');
     // push the new catchphrase to the catchphrase array in state
-
+    const newCatchphrase = catchphraseInput.value;
     // clear out the form input's value so it's empty to the user
-   
+    catchphrases.push(newCatchphrase);
     // update the dom to show the new catchphrases (refactor to/call displayCatchphrases to do this work)
-
+    catchphraseInput.value = '';
+    displayCatchphrases();
 });
 
 function displayStats() {
```

Comment:
For this function, rather than returning this line, you will want to set reportEl.textContent = this line. Your template literal is great, but it is not displaying in the app because you are not injecting it into an DOM element.